tests/classes.py

```diff
@@ -26,10 +26,10 @@ def setUp(self):
         if TEST_TOKEN:
             self.client.login(token=TEST_TOKEN)
 
-        self.recorder = Betamax(session=self.client.session)
-        self.recorder.use_cassette(self.cassette_name)
-        self.recorder.start()
+        # self.recorder = Betamax(session=self.client.session)
```

Comment:
I guess this is temporary turned off. Lets make sure this is not in the final commit